deployments/CRD/KubeArmorAlertRule.yaml

```diff
@@ -0,0 +1,128 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
```

Comment:
How is this CRD generated?

Reply:
what do you mean by how it is generated? I create it manually

Reply:
You probably used kube-builder right? It would be generated somehow right? Like we have https://github.com/kubearmor/KubeArmor/tree/main/pkg which have respective custom resources and have their resources generated their?

Reply:
no, I dont use kube-builder, I literally type the KubeArmorAlertRule by myself. What tools should I use for this? Do you recommend kube-builder?

Reply:
It would help you generate clientset too so yes.